src/Core/src/MauiContextExtensions.cs

```diff
@@ -65,12 +65,27 @@ public static IMauiContext MakeWindowScope(this IMauiContext mauiContext, Native
 			scopedContext.AddSpecific(new NavigationRootManager(platformWindow));
 #endif
 
+			// Initialize any window-scoped services, for example the window dispatchers and animation tickers
+			scopedContext.InitializeScopedServices();
```

Comment:
yea, this is what I did in the original-original PR.
I was just worried about shifting this behavior in NET8 
Which is why I created the tracking PR here https://github.com/dotnet/maui/issues/19591

Reply:
Not sure we can keep this in the same place as it would instantiate/touch the scopedservices outside the window scope - this actually threw for me in my tests when I enabled the scope validation.

Reply:
Yea, I got around this in a somewhat hacky way :-( 

So, I modify the `WinUI` level workaround to work at the app level
https://github.com/dotnet/maui/pull/19593/files#diff-d6d1eff30f175dfccc3a154b3f416f8c8fbd5f641707b299f1e4647866df90b9R32

And then added this really ugly code to init the dispatcher for each window
https://github.com/dotnet/maui/pull/19593/files#diff-500f3422fdff10304272c2e7985ec98941543f788bef70ac1fb1b23776bb1104R80 (which I'm realizing I could have done better with a new internal interface)

The main motivation being that, it feels somewhat scary to change the behavior of `IMauiInitializeScopedService` in an SR. Like, this change will make it so `IMauiInitializeScopedService` runs against the window scoped services vs the app service. Which is 100 percent the right behavior :-) and if it breaks anyone, they should change to using `IMauiInitializeService`

 
Would it be safer to just introduce a new internal interface for now? `IMauiInitializeWindowService` or something like that.

And then in NET9 we deprecate `IMauiInitializeScopedService`? 

Or do we just want to say lets fix it :-)

Reply:
Not sure many things are scoped, so we might as well fix it and see if breaks anyone in the nightly.